rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/rename_class_with_same_name_but_different_namespace_multiple_use_statements.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;
+
+use Rector\Core\Rector\AbstractRector;
+use Rector\Tests\Renaming\Rector\Name\RenameClassRector\Source\FirstNamespace\SomeServiceClass;
+use Doctrine\Instantiator\Instantiator;
```

Comment:
What is this use statement for?

Reply:
As i was investigating this i thought it has something to do with having multiple use statements. But i think this is not true anymore. I am gonna delete this test fixture here.